scheduler/docs/scheduler-rest-api.asc

```diff
@@ -69,6 +69,7 @@ The request body is list of job maps with each entry containing a map of the fol
 |JSON Key | Type | Description
 |`uuid` | string | a user provided UUID for tracking and referring to the job.
 |`command` | string | the command to run
+|`cook_executor` | boolean | Flag to force use of the Cook executor, defaults to false.
```

Comment:
@pschorf @sradack @wyegelwel how do you feel about adding a new field `executor=cook|mesos` and leaving the `custom-executor` flag for backwards compatibility.